### PR TITLE
Add production application checklist

### PR DESCRIPTION
**Pull request by tpendragon**
Special thanks to all of IT for their contributions to this document.

### CODE REVIEW

File: norms/production_application_checklist.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+
+### Infrastructure as Code
+
+#### Reasoning
+
+Defining our infrastructure as code lets us easily reproduce an environment if it needs to be replaced or upgraded. Manual intervention on machines makes them more fragile, so we avoid that with infrastructure as code technology. An example is Ansible. If there is no way to define the infrastructure as code, then the steps to get to the desired state should be documented as a run book.
+
+### One Application Per Server
+
+#### Reasoning
```
Comment (escowles) [inline norms/production_application_checklist.md:28]: I'm not sure the "Reasoning" header adds a lot in cases like this where it's the whole section. Maybe skip it unless there is a need to differentiate the reasoning from something else?

**Review by escowles — APPROVED** [review]
👍🏻 This looks good to me

File: norms/production_application_checklist.md (hunk 1 above)
Comment (tpendragon) [reply norms/production_application_checklist.md:28]: I just liked the symmetry.

Comment (escowles) [reply norms/production_application_checklist.md:28]: That's fine — it's a stylistic thing that's not really important.

**Review by carolyncole — APPROVED** [review]
Nice work @maxkadel and @tpendragon!  My only thought is that when we have time later it would be lovely to have examples of tools linked so everyone doesn't have to dig them up.

**Review by hackartisan — APPROVED** [review]
👍🏻

**Review by kayiwa — APPROVED** [review]
OK! Thanks @maxkadel and @tpendragon

**Review by kevinreiss — APPROVED** [review]
Thanks for your work on this @maxkadel and @tpendragon.

**Pull request merged**